fuel-database/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "fuel-database"
+version = "0.14.1"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+categories = ["cryptography::cryptocurrencies"]
+edition = "2021"
+homepage = "https://fuel.network/"
+keywords = ["blockchain", "cryptocurrencies", "fuel-database", "fuel-core"]
+license = "BUSL-1.1"
+repository = "https://github.com/FuelLabs/fuel-core"
+description = "Fuel database contains primitives and abstractions to work with the database."
+
+[dependencies]
+anyhow = "1.0"
+bincode = "1.3"
+enum-iterator = "1.2"
+fuel-core-interfaces = { path = "../fuel-core-interfaces", version = "0.14.1", features = ["serde"] }
+fuel-metrics = { path = "../fuel-metrics", version = "0.14.1", optional = true }
+itertools = "0.10"
+rocksdb = { version = "0.19", default-features = false, features = [
+    "lz4",
+    "multi-threaded-cf"
+], optional = true }
+serde = { version = "1.0", features = ["derive"] }
+strum = "0.24"
+strum_macros = "0.24"
+tempfile = "3.3"
+thiserror = "1.0"
+
+[dev-dependencies]
+rand = "0.8"
+
+[features]
+default = []
```

Comment:
Default features on `fuel-core` include both `metrics` and `rocksdb`, wouldn't this then be a behavior change?

Reply:
No, because `fuel-core` still enables those features by default=)